huaweicloud/resource_huaweicloud_cce_cluster_v3_test.go

```diff
@@ -209,3 +231,35 @@ resource "huaweicloud_cce_cluster_v3" "test" {
 }
 `, testAccCCEClusterV3_Base(rName), rName)
 }
+
+func testAccCCEClusterV3_withEpsId(rName string) string {
+	return fmt.Sprintf(`
+
+resource "huaweicloud_vpc_v1" "test" {
+  name = "%s"
+  cidr = "192.168.0.0/16"
+  enterprise_project_id = "%s"
+}
+	
+resource "huaweicloud_vpc_subnet_v1" "test" {
+  name          = "%s"
+  cidr          = "192.168.0.0/16"
+  gateway_ip    = "192.168.0.1"
+	
+  //dns is required for cce node installing
+  primary_dns   = "100.125.1.250"
+  secondary_dns = "100.125.21.250"
+  vpc_id        = huaweicloud_vpc_v1.test.id
+}
```

Comment:
this can be replaced with  testAccCCEClusterV3_Base(rName) as L232

Reply:
fixed